src/components/DateTimePicker.tsx

```diff
@@ -1,29 +1,33 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import React, { useCallback, useEffect, useMemo, useState } from 'react'
-import { Calendar, TimeInput, TimeInputValue, CalendarProps } from 'gfx-component-lib'
+import { Calendar, TimeInput, TimeInputValue, CalendarProps, Button } from 'gfx-component-lib'
 import dayjs from 'dayjs'
+import useBreakPoint from '@/hooks/useBreakPoint'
 
 export type DateTimePickerProps = {
   value: dayjs.Dayjs
   onChange: (date: dayjs.Dayjs) => void
+  setDrawerOpen?: (b: boolean) => void
```

Comment:
Kind of thinking, should weextract the button at the bottom and pass it as something like "FooterChildren" so the DateTimePicker doesnt become a single purpose component; or what do you think?